pubs/config/spec.py

```diff
@@ -99,6 +99,11 @@
 # command = !pubs list -k | wc -l
 # description = lists number of pubs in repo
 
+# To use commas in the description, wrap them in a "" string. For example:
+# [[[hellocount]]]
+# command = !pubs list -k | wc -l; echo "hello!"
```

Comment:
Minor comment here: for the sake of conciseness, I would suggest keeping only the first and last line of your change (no need for a new fake section and a new fake comment).

Reply:
Done :)